ui/src/components/chat/compose.rs

```diff
@@ -343,25 +343,34 @@ fn get_chatbar(cx: Scope<ComposeProps>) -> Element {
             )),
             with_replying_to: data.map(|data| {
                 let active_chat = data.active_chat.clone();
+                let participants = build_participants(&data.active_chat.participants.clone());
+
                 cx.render(rsx!(
-                    active_chat.clone().replying_to.map(|msg| rsx!(
+                    active_chat.clone().replying_to.map(|msg|
+                    {
+                    let our_did = state.read().account.identity.did_key();
+                    let their_did = msg.sender();
+                    let (platform, status) = if our_did == their_did {
+                        (participants[1].platform, participants[1].status)
+                    } else {
+                        (participants[0].platform, participants[0].status)
+                    };
+
+                    rsx!(
                         Reply {
                             label: get_local_text("messages.replying"),
-                            remote: {
-                                let our_did = state.read().account.identity.did_key();
-                                let their_did = msg.sender();
-                                our_did != their_did
-                            },
+                            remote: our_did != their_did,
```

Comment:
`remote` could become `our_did != msg.sender()` if the variable for `their_did` is done away with